.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: Enhancement Request
 about: Suggest an enhancement to the gateway-api project
-labels: kind/feature
+labels: kind/gep
```

Comment:
🤔  It seems like we won't want every feature request to necessarily be a GEP. IMO, issues should probably only transition to `kind/gep` once we have a GEP PR open (or about to be), and `kind/feature` can continue to be used to represent feature requests that are earlier along in their lifecycle. Not intended to be a blocking comment here, just want to see how you or others view the ideal distinction between these.